pypit/arparse.py

```diff
@@ -992,6 +992,7 @@ def reduce_flatfield_params(self, v):
         specified by the 'reduce flatfield method' keyword:
 
         polyscan:  [Polynomial order, Number of pixels, Number of repeats]
+        bspline:   [Number of pixels in the dispersion direction between each knot]
```

Comment:
This is in fact 1D, because I multiply this argument by the length of the slit (in pixels).